src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -31,39 +31,44 @@
   /**
    * The target velocity of the NEO Brushless Motor.
    */
-  private final double shooterSetpoint = 1.0;
+  public final double shooterSetpoint = 1.0;
   /**
    * The Proportial Gain of the SparkMAX PIDF controller The weight of the
    * proportional path against the differential and integral paths is controlled
    * by this value.
    */
-  private final double kP = 1;
+  private final double kP = 0.1;
```

Comment:
Still too big.  Review the Spark/NEO PID constants from last year or from the Rev sample code.  You'll find that P should be much smaller.

Reply:
Done.